### ISSUE TITLE
Tensor must have a shape comprised of positive integers but got shape [1917,]

### ISSUE THREAD

**Issue by mingsoohea** [cloud-annotations/object-detection-react#29]
I followed cloud.annotations.ai to train a custom model for tensorflow.js.
After cloning this project and copy the model_web to the public folder, I `run npm install` then `npm run start.`

The local page is showed with an error when detectFromVideoFrame:
`Error: Tensor must have a shape comprised of positive integers but got shape [1917,].
    at g (index.esm.js:17)
    at index.esm.js:17
    at Array.forEach (<anonymous>)
    at Q (index.esm.js:17)
    at kn (index.esm.js:17)
    at Object.Dn [as tensor2d] (index.esm.js:17)
    at index.esm.js:500
    at index.esm.js:17
    at t.scopedRun (index.esm.js:17)
    at t.tidy (index.esm.js:17)
    at Object.Ke [as tidy] (index.esm.js:17)
    at runObjectDetectionPrediction (index.esm.js:499)
    at async Object.detect (index.esm.js:556)`

After digging a bit into the source code, I found this strange:
![Untitled](https://user-images.githubusercontent.com/62318760/100336164-68bcb000-3008-11eb-8141-85059a332204.png) 

![Untitled1](https://user-images.githubusercontent.com/62318760/100336170-6a867380-3008-11eb-9a0d-791ec3626449.png)

![Untitled2](https://user-images.githubusercontent.com/62318760/100336588-f3051400-3008-11eb-8d04-1128c36af6f1.png)

![Untitled3](https://user-images.githubusercontent.com/62318760/100336988-69a21180-3009-11eb-95a8-682f323c1aca.png)

As you see, the scores and boxes have been swapped together. Is it intended?

**Comment by mingsoohea** [comment]
@bourdakos1

**Comment by bourdakos1** [comment]
This normally means you didn’t have enough training data

**Comment by mingsoohea** [comment]
I used 200 images to annotate on cloud.annotations.ai.
How many images should I use at least?
From the last screenshot, result[1].shape only have 3 components [1, 1917, 1] but the code tries to get result[1].shape[3]
Is it make sense?

**Comment by bourdakos1** [comment]
200 hundred images might be enough, but it more depends on how many labels and how many objects per label

**Comment by mingsoohea** [comment]
1 label for all
1 object per image only.

**Comment by bourdakos1** [comment]
Try adding a second label

**Comment by Roger92** [comment]
Did it help to add more images or an additional lable? Because i have the same error :D

**Comment by sebaspv** [comment]
I'm having the same error. Did adding more images help? I have 3 labels and 200 images for each of them

**Comment by menuRivera** [comment]
Does anybody have a working model_web so I can make some tests?
